examples/package.json

```diff
@@ -9,7 +9,7 @@
 		"url": "git://github.com/redislabs/redisgraph.js.git"
 	},
 	"dependencies": {
-		"redisgraph.js": "^1.2.0"
+		"redisgraph.js": "github:RedisGraph/redisgraph.js"
```

Comment:
it might be an issue if someone will copy it, you might want to set a relative path